### ISSUE TITLE
PR code review guidelines

### ISSUE THREAD

**Issue by rborn** [react-native-community/.github#8]
We should mention in the guidelines that anyone can review the PRs (to free some load from the maintainers) but int he same time the release and merge procedures are restricted so we can keep the builds in a working state.

**Comment by kelset** [comment]
@orta posted an RFC to add Peril to the org [here](https://github.com/react-native-community/discussions-and-proposals/pull/116), maybe it's something that we can use to enforce said rules? (once we define them?)

**Comment by rborn** [comment]
@kelset @orta didn't know about peril, I'll have a deeper look but it looks good at a first glance 🤗

**Comment by orta** [comment]
I think most of the these [discussion issues](https://github.com/react-native-community/meta/issues?q=is%3Aissue+is%3Aopen+label%3Adiscussion) could be verified with a daily task in Peril

**Comment by kelset** [comment]
For context, here's Peril's repo https://github.com/danger/peril#i-want-to-use-peril

Btw @orta in that section it says that basically we'd need to self-host. Any suggestions on that?

Should we follow this? https://github.com/danger/peril/blob/master/docs/setup_for_org.md

**Comment by orta** [comment]
Nah, don't self-host - run on Peril staging. I can hook that up 👍

**Comment by orta** [comment]
@kelset can you make a repo called peril-settings for this org? Then I can do the rest I think
